src/main/scala/viper/silver/ast/utility/QuantifiedPermissions.scala

```diff
@@ -91,44 +137,11 @@ object QuantifiedPermissions {
     } toSet) flatten

```

Comment:
I know that this was not touched in this PR, but is there any reason why `quantifiedMagicWands` is implemented so differently than `quantifiedFields` and `quantifiedPredicates`? In particular, I wonder if this method should also call `collectInDependencies` somehow

Reply:
Good question, I have no idea. I added a TODO in the code and wrote it down on my list of things to check at some point.